src/main/frontend/components/EditVenue.js

```diff
@@ -0,0 +1,232 @@
+import React, { useState, useEffect } from "react";
+import { Redirect } from "react-router";
+import { useLocation } from "react-router-dom"
+import _ from "lodash";
+
+import StateDropdown from "./StateDropdown";
+import ErrorList from "./ErrorList";
+
+const EditVenue = (props) => {
+  let location = useLocation();
+  const [formPayload, setFormPayload] = useState({
+    name: "",
+    address: "",
+    city: "",
+    state: "",
+    zipCode: "",
+    capacity: "",
+    description: "",
+    imgUrl: "",
+    phoneNumber: ""
+  });
+
+  const [errors, setErrors] = useState({});
+  const [shouldRedirect, setShouldRedirect] = useState(false);
+  const [venueId, setVenueId] = useState(null);
+
+  const fetchVenue = async () => {
+    try {
+      const id = props.match.params.id;
+      setVenueId(id);
+      const response = await fetch(`/api/v1/venues/${id}`);
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const responseBody = await response.json();
+      setFormPayload(responseBody.venue);
```

Comment:
It looks like the `responseBody.venue` will also return with the correct `id` if you want to reference that

Reply:
Noted. Since I removed the venueID state, this is no longer needed.